portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -1,18 +1,29 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
+
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.gson.Gson;
+
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
-
+  private ArrayList<String> facts = new ArrayList<String>();
```

Comment:
Since the same facts are always returned, you can probably use the immutable `List` datastructure in Java and just initialize it here.

```
List<String> facts = Arrays.asList("a", "b", "c");
```

That way you can remove all the `add` calls below and the `if`.